upgrade/UPGRADE-v9.0.0-dev.md

```diff
@@ -510,6 +510,102 @@ There you can find links to upgrade notes for other versions too.
         +   ): ProductFilterCountData;
         ```
 
+- add cart detail on hover ([#1565](https://github.com/shopsys/shopsys/pull/1565))
+  
+  - you can skip this task if you have your custom design
+  - Add new file [src/Resources/styles/front/common/layout/header/cart-detail.less](https://github.com/shopsys/shopsys/blob/9.0/project-base/src/Resources/styles/front/common/layout/header/cart-detail.less)
+  - Update your src/Resources/styles/front/common/layout/header/cart.less [diff](https://github.com/shopsys/shopsys/pull/1565/files#diff-bc98fd209f1c026440cbf870086beece)
```

Comment:
_nitpick_
I suggest using formulation like "Update your <file> like in the [diff]" (if you agree, please use it everywhere)